cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -147,22 +148,23 @@ func (spm *KubeStaticPodPathManager) BackupEtcdDir() string {
 
 // CleanupDirs cleans up all temporary directories except those the user has requested to keep around
 func (spm *KubeStaticPodPathManager) CleanupDirs() error {
-	if err := os.RemoveAll(spm.TempManifestDir()); err != nil {
-		return err
+	var err []error
```

Comment:
this could be:
```
var errlist []error
```

same as:
https://github.com/kubernetes/kubernetes/blob/4992402fa5bee9f7ac6fe931d101d362a4295cea/staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go#L41